docs/windows.md

```diff
@@ -59,7 +59,43 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (containerd based runtimes)
+
+First install antrea (v0.13.0+ is required for containerd).
+
+```bash
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.13.0/antrea.yml
+```
+
+Then, you can run the following commands. This will setup antrea as a Windows service.
+
+```bash
+$TAG = "v0.13.0"
+
+curl.exe -LO "https://raw.githubusercontent.com/vmware-tanzu/antrea/${TAG}/hack/windows/Helper.psm1"
+curl.exe -LO "https://github.com/vmware-tanzu/antrea/releases/download/${TAG}/antrea-agent-windows-x86_64.exe"
+
+mv antrea-agent-windows-x86_64.exe c:/k/antrea/bin/antrea-agent.exe
+Import-Module ./helper.psm1
+Install-AntreaAgent -KubernetesVersion "v1.20.2" -KubernetesHome "c:/k" -KubeConfig "C:/etc/kubernetes/kubelet.conf" -AntreaVersion "${TAG}" -AntreaHome "c:/k/antrea"
```

Comment:
KubeConfig is important here as well, we've prototyped with this being the kubelet.conf but by default the node doesn't have access to everything it needs, is there a way that we can allow the node config to do what it needs to or is that a security problem and can we think of a way around it (avoiding manually creating the kubeconfig)

Reply:
Thank you @antoninbas and @perithompson . I am investigating these issues. Yes, on my testbed, `kubelet.conf` is not enough to get secret in `Install-AntreaAgent()`.

Reply:
Discussed with @ruicao93. As for the K8s version, there's no requirement. So here `v1.20.4` is just an example, users should keep it consistent with K8s version on other nodes.
As for kubeconfig, it is users' responsibility to provide a valid kubeconfig. Whether using `admin.conf` or a customed one, it's  up to the user.

Reply:
understood, it's just incredibly difficult to automate this at node creation time so I was wondering if there was a way around this to help

Reply:
I have mentioned that kubeconfig can be copied from `/etc/kubernetes/admin.conf` on ControlPlane node. I suppose it is enough for users.

Reply:
An alternative could be to provide a script to generate all the required kubeconfig files (2 for antrea-agent and 1 for kube-proxy) using the "admin" kubeconfig. This process would need to be done only once per cluster instead of on every Node. We could then use a DaemonSet to distribute and install the generated kubeconfig files to each Windows Node in the cluster. As a new Node joins the cluster, the files would be installed automatically. But maybe that's an overkill...

It seems that currently we recommend running kube-proxy with the "admin" kubeconfig (`/etc/kubernetes/admin.conf` copied from the control-plane mode), which seems like too many permissions.

Reply:
I agree it is an overkill. A simple script is enough to distribute kubeconfig to nodes. Then how about this? Admin kubeconfig is used to `Install-AntreaAgent` because such permission is required to get secret and kubelet one is used to `install kube-proxy`. I have updated this doc.

Reply:
In the context of capi, when you create a node, you only have control of actions that happen on the host as an administrator, so in theory I guess you could get it from another host from the cluster but securing that is a nightmare too!